app/models/relationship.rb

```diff
@@ -0,0 +1,6 @@
+class Relationship < ApplicationRecord
+  belongs_to :follower, class_name: "User"
```

Comment:
Sao ở đây lại cần khai báo class_name 
```suggestion
  belongs_to :follower, class_name: User.name 
```

Reply:
Để chỉ rõ quan hệ  với model nào nếu trong TH này nếu không chỉ định rails sẽ hiểu là quan hệ với model Follower mà ko phải là model User